compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Window.desktop.kt

```diff
@@ -416,6 +418,7 @@ fun Window(
         update = {
             it.compositionLocalContext = compositionLocalContext
             it.exceptionHandler = windowExceptionHandlerFactory.exceptionHandler(it)
+            it.scene.mainOwner?.layoutDirection = layoutDirection
```

Comment:
```suggestion
            it.layoutDirection = layoutDirection
```
The more right approach will be to create public properties `ComposeWindow.layoutDirection` and `ComposeScene.layoutDirection` (and remove `Platform.layoutDirection`). We can mark them experimental for now

Reply:
Who will be the source-of-truth? Right now each `Owner` defines its own `layoutDirection`.

Reply:
> the source-of-truth

The root (application), unless some child composition overrides it for all children.

In our case, ComposeWindow/ComposeScene sets the direction for the main composition, but it doesn't prohibits overriding it by the user with CompositionLocalProvider for child compositions that should affect child owners (Popup/Dialog).

Reply:
Doing in https://github.com/JetBrains/compose-multiplatform-core/pull/780